config/core.extension.yml

```diff
@@ -52,7 +52,6 @@ module:
   path: 0
   path_alias: 0
   pcb: 0
-  quickedit: 0
```

Comment:
@attiks FYI I disabled Quicklinks module
In case you care :) It adds markup and that gets in the way when I'm trying to look at the output with dev tools. I can re-enable nearer the end of the project. Maybe the content editors use it.